docs/DeveloperGuide.md

```diff
@@ -289,16 +289,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `CCACommander` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: UC01 - Delete a member**
+<ins>**Use case: UC01 - Create a member**</ins>
+
+**Guarantees: MSS -> Entered member will be added.**
+
+**MSS**
+
+1.  User enters command to create a member.
+2.  CCACommander adds that member to the member list and displays the member added.
+
+    Use case ends.
+
+**Extensions**
+* 1a. User uses the command incorrectly.
+
+    * 1a1. CCACommander shows an error message.
+
+      Use case ends.
+
+<ins>**Use case: UC02 - Delete a member**</ins>
 
 **Guarantees: MSS -> Specified member will be deleted.**
 
 **MSS**
 
-1. User requests to <ins>list all members</ins> (UC02)
-2. CCACommander shows a list of all members
-3. User requests to delete a specific member in the list of all members
-4. CCACommander deletes the specific member
+1. User requests to <ins>list all members</ins> (UC02).
```

Comment:
This 'list all members' use case will be UC03, due to a change in ordering as the 'create member' use case is added.

Reply:
Updated